redis/src/caching/config.rs

```diff
@@ -0,0 +1,61 @@
+use std::num::NonZeroUsize;
+use std::time::Duration;
+
+/// Defines the behavior of the cache regarding which commands should be cached.
+#[derive(Clone, PartialEq, Debug, Copy)]
+pub enum CacheMode {
+    /// Try to cache every command. This is equivalent to calling [crate::Cmd::set_cache_config]/[crate::Pipeline::set_cache_config] on every command.
+    /// Disabling individual commands for caching with this [CacheMode::All] will not work.
+    All,
+    /// Only try to cache commands which explicitly enabled caching.
```

Comment:
Only cache commands* that are explicitly marked by the user for caching?

or "keys from commands"